File: measure_time.py
import time

start_time = time.time()

result = 0
for i in range(1000000):
    result += i * i  # Simple computation

end_time = time.time()
print(f"Execution Time: {end_time - start_time:.6f} seconds")

# python3 measure_time.py

File: measure_time.cpp
#include <iostream>
#include <chrono>

int main() {
    auto start = std::chrono::high_resolution_clock::now();

    volatile long long result = 0;  // volatile prevents optimization
    for (long long i = 0; i < 1000000; i++) {
        result += i * i;  // Simple computation
    }

    auto end = std::chrono::high_resolution_clock::now();
    std::chrono::duration<double> execution_time = end - start;

    std::cout << "Execution Time: " << execution_time.count() << " seconds" << std::endl;
    return 0;
}

/*
g++ -std=c++17 measure_time.cpp -o measure_time
./measure_time
*/
